scripts/release_ppa.sh

```diff
@@ -9,13 +9,17 @@
 ## If the given branch is "release", the resulting package will be uploaded to
 ## ethereum/ethereum PPA, or ethereum/ethereum-dev PPA otherwise.
 ##
-## The gnupg key for "builds@ethereum.org" has to be present in order to sign
-## the package.
-##
 ## It will clone the Solidity git from github, determine the version,
 ## create a source archive and push it to the ubuntu ppa servers.
 ##
-## This requires the following entries in /etc/dput.cf:
+## To interact with launchpad, you need to set the variables $LAUNCHPAD_EMAIL
+## and $LAUNCHPAD_KEYID in the file $PROJECTROOT/.release_ppa_auth to your launchpad email and pgp keyid.
```

Comment:
We don't really have this variable so it may be a bit confusing. Maybe replace it with text?
```suggestion
## and $LAUNCHPAD_KEYID in the file .release_ppa_auth in the root directory of the project to your launchpad email and pgp keyid.
```